### PR TITLE
VaR models clean up

### PR DESCRIPTION
**Pull request by leafyoung**
This removes all asOf field from VaR models and merged the back-end of Compound/Aave VaR to use the `finance.var-portfolio-historical`.

Previous PR:
https://github.com/credmark/credmark-models-py/pull/86

### CODE REVIEW

File: models/credmark/algorithms/value_at_risk/dto.py [hunk 1]
```diff
@@ -1,4 +1,3 @@
-from datetime import date
```
Comment (newtectonics) [inline models/credmark/algorithms/value_at_risk/dto.py:1]: love this

File: models/credmark/algorithms/value_at_risk/var.py [hunk 2]
```diff
@@ -1,15 +1,63 @@
...
+                description='Calculate VaR based on input portfolio',
+                input=PortfolioVaRInput,
+                output=dict)
+class VaRPortfolio(Model):
+    def run(self, input: PortfolioVaRInput) -> dict:
+        portfolio = input.portfolio
+
+        pls = []
+        pl_assets = set()
+        for pos in portfolio:
```
Comment (newtectonics) [inline models/credmark/algorithms/value_at_risk/var.py:29]: can we expand variables? I know it's longer, but it makes it easier to understand

Comment (newtectonics) [reply models/credmark/algorithms/value_at_risk/var.py:29]: so `for position in portfolio`

**Review by newtectonics — APPROVED** [review]
I think some of this might be a merge, but I love less exposure to datetime (which has potential to upend determinism) and simplifications of certain models in the finance namespace.

File: models/credmark/algorithms/value_at_risk/var.py (hunk 2 above)
Comment (leafyoung) [reply models/credmark/algorithms/value_at_risk/var.py]: ok. renamed.

**Pull request merged**